packages/front-end/services/features.ts

```diff
@@ -163,10 +166,12 @@ export function useFeaturesList(withProject = true) {
 
   const { data, error, mutate } = useApi<{
     features: FeatureInterface[];
+    linkedExperiments: ExperimentInterface[];
```

Comment:
This is really `ExperimentInterfaceStringDates[]` since Date objects are serialized in API responses